meeting_minutes/220825_SRAIF_TC_65.md

```diff
@@ -0,0 +1,158 @@
+|:-----------|:----------|:------------------|:------------------------|
+| David      | Keaton    | Individual        | Chair                   |
+| Jeff       | Williams  | Contrast Security | Voting Member           |
+| Gerald     | Sullivan  | Micro Focus       | Voting Member           |
+| Johnathan  | Gilday    | Contrast Security | Member                  |
+| Mary       | Martin    | Microsoft         | Voting Member           |
+| Nathan     | Baird     | Microsoft         | Voting Member           |
+| Paul       | Anderson  | GrammaTech, Inc.  | Voting Member           |
+| Pavel      | Mikula    | Sonar             | Voting Member           |
+| Thanassis  | Avgerinos | ForAllSecure Inc  | Voting Member           |
+| Yekaterina | O'Neil    | Micro Focus       | Voting member           |
+
+## 1.3 Procedures for this meeting (Co-Chair David)
+
+## 1.4 Approval of agenda (Co-Chair David)
+
+* https://www.oasis-open.org/committees/download.php/70286/agenda_20220825.html
+
+Agenda was approved
+
+## 1.5 Approval of previous minutes (Co-Chair David)
+
+* [Minutes of 2022-08-11_Meeting #64](https://www.oasis-open.org/committees/document.php?document_id=70287&wg_abbrev=sarif)
+
+Minutes were approved
+
+## 1.6 Review of action items and resolutions (Co-Chair David)
+
+* ACTION on David to keep pushing to close on the wikipedia page
+  * ONGOING 
+* ACTION on Mary to continue to develop fixes scenario
+  * ONGOING 
+* ACTION on Michael to rename from master to main (URL)
+  * ONGOING 
+* ACTION on Thanassis to present runtime-specific data, not August 11 but next one, the 25th  
+  * ONGOING
+* ACTION on Michael to provide additional references for the wikipedia
+  * ONGOING
+* ACTION on Jeff to provide an initial seed page at github to foster the future roadmap / extension discussion
+  * ONGOING
+* ACTION on Stefan to create a list for plausible potential as a github page for boosting the SBOM/other standards discussion
+  * ONGOING
+* ACTION on Michael to invite for a subgroup meeting mid next week or thereafter to kick-off the evaluation of potential other standards to align with in the security realm
+  * COMPLETE
+
+## 1.7 Identification of SARIF TC voting members (Co-Chair David)
+
+### 1.7.1 Prospective members attending their first meeting
+
+### 1.7.2 Members attaining voting rights at the end of this meeting
+
+* Johnathan Gilday will receive voting privileges at the end of this meeting
+
+### 1.7.3 Members losing voting rights if they have not joined this meeting by the time it ends
+
+* Chris Meyer will lose voting privileges
+* Chris Wysopal will lose voting privileges
+
+### 1.7.4 Members who previously lost voting rights who are attending this meeting
+
+### 1.7.5 Members who have declared a leave of absence
+
+# 2. Future Meetings
+
+## 2.1 Future meeting schedule (Co-Chair Keaton)
+
+- Scgeduled Teleconferences (Thursdays at 08:00 PT / 15:00 UTC for 1.5 hours)
```

Comment:
Scgeduled -> Scheduled

Reply:
It reads so much more interesting though ... thanks.